prometheus-exporters/install-exporter-amd64.sh

```diff
@@ -374,6 +412,21 @@ EOF
 EOF
   fi
 
+  if [ "$EXPORTER" == "nginx" ]; then
+    cat << EOF > /etc/opsverse/targets/${EXPORTER}-exporter.json
+[
+  {
+    "labels": {
+      "job": "integrations/nginx-prometheus-exporter"
+    },
+    "targets": [
+      "localhost:80"
```

Comment:
This port is wrong. `80` might be the port where nginx is listening, but the `nginx-exporter`, by default, listens on `9113`. So, that should be the port here.